<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsGdalCloudConnectionDialog</class>
 <widget class="QDialog" name="QgsGdalCloudConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cloud Storage Connection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QLineEdit" name="mEditName">
     <property name="toolTip">
      <string>Name of the new connection</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="mCredentialsGroupBox">
     <property name="title">
      <string>Credentials</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="mBucket"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="mKey"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelBucket">
     <property name="text">
      <string>Bucket or container</string>
     </property>
     <property name="buddy">
      <cstring>mBucket</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelKey">
     <property name="text">
      <string>Object key</string>
     </property>
     <property name="buddy">
      <cstring>mKey</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsGdalCloudConnectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsGdalCloudConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
